olm-catalog/serverless-operator/project.yaml

```diff
@@ -64,9 +64,9 @@ dependencies:
     operator: 1.11.0
     # Previous versions required for downgrade testing
     previous:
-        serving: knative-v1.10
-        eventing: knative-v1.10
-        eventing_kafka_broker: knative-v1.10
+        serving: knative-v1.11
```

Comment:
Uh, previous, as in `1.32.0` is "previous to this?

That's a weak / difficult knob, IMO. But likely subject for a different discussion